packages/blade/src/components/Box/Box.tsx

```diff
@@ -123,6 +124,9 @@ const makeBoxProps = (props: BoxProps): KeysRequired<Omit<BoxProps, 'testID' | '
     backgroundOrigin: props.backgroundOrigin,
     backgroundRepeat: props.backgroundRepeat,
     elevation: props.elevation,
+    transform: props.transform,
+    transformOrigin: props.transformOrigin,
+    clipPath: props.clipPath,
```

Comment:
So all props that we add here need to be handled in BaseBox.

The way it works is, BaseBox is superset of Box and styled-props. Everything we add here needs to be handled in BaseBox.

You can checkout how we handle it here - 

https://github.com/razorpay/blade/blob/d84a656f178bf25cf44bd84038cbc3481e58126e/packages/blade/src/components/Box/BaseBox/baseBoxStyles.ts#L207-L211

This makes the props work. It handles there types and also makes the responsiveness work with props. E.g. - `pointerEvents={{ base: 'none', m: 'all' }}`

Reply:
Offtopic: Looks like we also missed adding `placeItems` here. Can you add it while adding other props?